src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/article/ArticlePageObject.java

```diff
@@ -309,9 +309,14 @@ public void verifyCommentText(String comment) {
     Assertion.assertStringContains(mostRecentComment.getText(), comment);
   }
 
-  public void verifyCommentVideo(String videoName) {
-    driver.findElement(By.cssSelector(videoInCommentsSelector.replace("%videoName%", videoName)));
-    PageObjectLogging.log("verifyCommentVideo", "video is visible in comments section", true);
+  public boolean verifyCommentVideo(String videoName) {
```

Comment:
This name is not clear for me. As far as I understand the code, this method checks if there is a comment that is made of video, so my proposition of the name is: isVideoCommentPresent
What do you think?